firstvoices/backend/admin/dictionary_admin.py

```diff
@@ -87,6 +105,7 @@ class PartsOfSpeechAdmin(BaseAdmin):
 admin.site.register(Note, HiddenBaseAdmin)
 admin.site.register(Acknowledgement, HiddenBaseAdmin)
 admin.site.register(DictionaryEntryLink, HiddenBaseAdmin)
+admin.site.register(DictionaryEntryCategory, HiddenBaseAdmin)
```

Comment:
I am getting a no `admin_link` error on dictionary entry admin pages because there is no reverse URL for the related characters admin page.

I think adding the following here should fix it:
`admin.site.register(DictionaryEntryRelatedCharacter, HiddenBaseAdmin)`

Reply:
Good catch, Nolan!